pkg/linguist/dev.go

```diff
@@ -76,22 +90,12 @@ func init() {
 				Remote: 8088,
 			},
 		},
-		securityContext: &model.SecurityContext{
-			RunAsUser:  &user1000,
-			RunAsGroup: &user1000,
-			FSGroup:    &user1000,
-		},
 	}
 
 	languageDefaults[maven] = languageDefault{
 		image:   "okteto/maven:latest",
+		path:    "/okteto",
 		command: []string{"bash"},
-		environment: []model.EnvVar{
```

Comment:
why are we removing this?

Reply:
they don't work. we switched to maven/gradle config files